inst/pages/example_solutions.qmd

```diff
@@ -81,7 +81,45 @@ These example solutions are related to exercises in
 #| label: taxonomy
 #| code-fold: true
 #| code-summary: "Show the solution"
-    
+
+#load example dataset
+library(mia)
+library(miaViz)
+data("Tengeler2020", package = "mia")
+tse <- Tengeler2020
+
+#Available taxonomic ranks
+taxonomyRanks(tse)
+
+#unique taxonomic features from class
+getUnique(tse, rank = 'Order')
+
+#Subsets the data and aligns the tree
+tse_sub <- tse[1:15, ]
+rowTree(tse_sub)$Nnode
+
+tse_sub <- subsetByLeaf(tse_sub, rowLeaf = rownames(tse_sub))
+rowTree(tse_sub)$Nnode
+
+#Creates plot for the phylogeny
+p1 <- plotRowTree(tse_sub) + theme(
+    panel.grid.major = element_line(color = "grey80", size = 0.5),
+    panel.grid.minor = element_line(color = "grey90", size = 0.25)
+)
```

Comment:
Let's make te examples as simple as possible, so theme could be removed as it is not essential